maintainers/maintainer-list.nix

```diff
@@ -5380,6 +5380,12 @@
     githubId = 221929;
     name = "Jean-Baptiste Giraudeau";
   };
+  jbuchermn = {
```

Comment:
This file should be in a separate commit with the title `maintainers: add myself`